tdms/include/globals.h

```diff
@@ -55,29 +49,27 @@ enum FieldComponents { Ex = 1, Ey, Ez, Hx, Hy, Hz };
 //  Enumerated constants
 // **********************
 
-enum ModeOfRun    { Pass1 , Pass2 };
-enum RCSType      { parallel , perpendicular };
+enum ModeOfRun { Pass1, Pass2 };
+enum RCSType { parallel, perpendicular };
 enum SolverMethod { PseudoSpectral, FiniteDifference };
 enum PreferredInterpolationMethods { BandLimited, Cubic };
 
 // **************************************
 //			Mathematical Constants
 // **************************************
 
-namespace tdms_math_constants
-{
-const double DCPI = 3.14159265358979323846;                                  // Pi
-const std::complex<double> IMAGINARY_UNIT = std::complex<double>(0.0, 1.0);  // Imaginary unit
-}
+namespace tdms_math_constants {
+  const double DCPI = 3.14159265358979323846;                                // Pi
+  const std::complex<double> IMAGINARY_UNIT = std::complex<double>(0.0, 1.0);// Imaginary unit
+}// namespace tdms_math_constants
 
 // **************************************
 //			Physical Constants
 // **************************************
 
-namespace tdms_phys_constants
-{
-const double EPSILON0 = 8.85400e-12;                         // free space electric permitivity
-const double MU0 = 4.0 * tdms_math_constants::DCPI * 1.0e-7; // free space magnetic permeability
-const double LIGHT_V = 1.0 / sqrt(EPSILON0 * MU0);           // free space light velocity
-const double Z0 = 376.734;                                   // free space inpedance
-}
+namespace tdms_phys_constants {

```

Comment:
Namespace indentation? 
Check with Google style and add something to `.clang-format`.